modules/mod_video/support/z_video_convert.erl

```diff
@@ -102,6 +103,7 @@ do_convert(QueuePath, State) ->
     end,
     mod_signal:emit({medium_update, [{id,State#state.id}]}, Context).
 
+-spec insert_movie(string(), #state{}) -> {ok, m_rsc:resource_id()} | {error, atom()}.
```

Comment:
The spec in line 106 uses a record, please define a type for the record and use that instead.